src/cms/views/analytics/analytics_overview.py

```diff
@@ -8,22 +8,22 @@
 
 @method_decorator(login_required, name="dispatch")
 @method_decorator(region_permission_required, name="dispatch")
-class SettingsView(TemplateView):
+class AnalyticsView(TemplateView):
     """
-    View for the region settings available to region managers
+    View to show all not-working links in the content.
     """
 
     #: The template to render (see :class:`~django.views.generic.base.TemplateResponseMixin`)
-    template_name = "settings/settings.html"
+    template_name = "analytics/analytics_overview.html"
     #: The context dict passed to the template (see :class:`~django.views.generic.base.ContextMixin`)
-    base_context = {"current_menu_item": "region_settings"}
+    base_context = {"current_menu_item": "analytics"}
 
     def get(self, request, *args, **kwargs):
```

Comment:
In this case, where no additional code is executed in the `get()`-method, we don't even need it. Just remove the `get()`-method altogether and let the rendering process be handled by the inbuilt Django functionality...
It's just important that `base_context` is renamed to `extra_context` so that Django includes it in the rendering process (see [ContextMixin.extra_context](https://docs.djangoproject.com/en/3.2/ref/class-based-views/mixins-simple/#django.views.generic.base.ContextMixin.extra_context))